client/src/components/SupportTools/SupportTools.tsx

```diff
@@ -203,12 +196,8 @@ const Organization = ({ organizationId }: { organizationId: string }) => {
       description: `Are you sure you want to delete organization ${name}?`,
       yesButtonLabel: 'Delete',
       onYesClick: async () => {
-        try {
-          await deleteOrganization.mutateAsync({})
-          toast.success(`Deleted organization ${name}`)
-        } catch (error) {
-          toast.error(error.message)
-        }
+        await deleteOrganization.mutateAsync()
+        toast.success(`Deleted organization ${name}`)
```

Comment:
in `onYesClick` handlers for `Confirm` dialogs, we need to use promises, because we want errors to bubble up to the dialog so that the dialog knows to stay open (vs closing on success)